dev/devicelab/lib/framework/apk_utils.dart

```diff
@@ -256,18 +256,26 @@ class FlutterProject {
   String get rootPath => path.join(parent.path, name);
   String get androidPath => path.join(rootPath, 'android');
   String get iosPath => path.join(rootPath, 'ios');
+  File get appBuildFile {
```

Comment:
Interesting that we have an independent `FlutterProject` here 
(independent of https://github.com/flutter/flutter/blob/88816add42666e2de88c4bffffa5fadf4cfcb9ab/packages/flutter_tools/lib/src/project.dart#L95).

I don't think I was aware of this

Reply:
Do you think it's worth to create an issue to track this?

(I'm not sure)

Reply:
I'll take a look to try to understand why it exists and make an issue if it doesn't seem to me like it should